charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,106 @@
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* -  the Ingest Support team member who assists labs contributing data to the HCA.
+
+*Data* - raw data files from an experiment such as [fastq files](https://en.wikipedia.org/wiki/FASTQ_format) for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - assay data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - assay data types which have a history of submission and downstream use in the DCP and for which there are already many datasets of this type represented in the platform
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor to submit one at a time. For example, a data contributor might upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*The PM team* - the team responsible for the oversight of the DCP as the product, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+## Objectives
+
+- Assisting data contributors uploading data and metadata to the DCP
+- Helping data contributors address errors with their data formatting and metadata before it can be submitted to the DCP. 
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to understand if new metadata fields are needed to represent their data
+- Proposing enhancements to the metadata schemas following the process defined by the Metadata Schema team
+- Working with the UX team to collect and provide feedback from data contributors to the Ingest Infrastructure team about how the infrastructure can be improved
+- Working with the UX team to ensure the ingest process provides a positive user experience to the data contributors
+
+## In-scope
+
+### For new data types
+
+- Working with data contributors to define a spreadsheet template to collect their metadata
+- Collecting metadata from data contributors in HCA ingest spreadsheet form
+- Collecting data files from data contributors
+- Reviewing collected data files and metadata
+- Submitting data files and metadata to the DCP on behalf of a data contributor
+
+### For established data types
+
+- Assisting data contributors uploading data files to the platform
+- Assisting data contributors using the ingest user interface to generate a metadata template spreadsheet
+- Assisting data contributors uploading metadata to the platform
+- Assisting data contributors who wish to use the ingest API to submit data
```

Comment:
Ingest API to match the style in Ingestion Service ...

I'm wondering whether we want to simply use DCP in place of _platform_